### PR TITLE
Add gems

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ services:
       - .:/task_manager:cached
     ports:
       - 3000:3000
-    volumes_from:
-      - bundle_cache
+    # volumes_from:
```
Comment (artempartos) [inline docker-compose.yml]: it is still ann issue?

Comment (moklidia) [reply docker-compose.yml]: It actually works now, just forgot to update.

File: test/test_helper.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 require 'rails/test_help'
 
 class ActiveSupport::TestCase
+	include FactoryBot::Syntax::Methods
```
Comment (artempartos) [inline test/test_helper.rb]: indent

Comment (moklidia) [reply test/test_helper.rb]: Fixed!

**Comment by artempartos** [comment]
👍

**Pull request merged**